### PR TITLE
TELCODOCS-1907 : RN Tuning CNI- ALLMULTI flag (GA, 4.14.z backport)

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1511,[TELCODOCS-1907](https://issues.redhat.com//browse/TELCODOCS-1907)]: Enabling ALLMULTI flag

Version(s): 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1907

Link to docs preview: https://80045--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-all-multicast

QE review:
- [x] QE has approved this change.

Additional information: RN for https://issues.redhat.com//browse/TELCODOCS-1907 now tested and verified on 4.14

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -861,6 +861,13 @@ Admin Network Policy is available as a Technology Preview feature. You can enabl
 
 With this release, the ability to create a MAC-VLAN, IP-VLAN, and VLAN subinterface based on a master interface in a container namespace is generally available. You can use this feature to create the master interfaces as part of the pod network configuration in a separate network attachment definition. You can then base the VLAN, MACVLAN or IPVLAN on this interface without knowing the network configuration of the node. For more information, see xref:../networking/multiple_networks/configuring-additional-network.html#nw-about-configuring-master-interface-container_configuring-additional-network[About configuring the master interface in the container network namespace].
 
+[id="ocp-4-14-all-multicast"]
+==== Supporting all-multicast mode
+
+OpenShift Container Platform release now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:867]:
```suggestion
{product-title} release now supports configuring the all-multicast mode by using the tuning CNI plugin. This update eliminates the need to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC), enhancing security by minimizing potential vulnerabilities for your pods.
```

**Review by tmalove — APPROVED** [review]

**Comment by adellape** [comment]
Looks like the xref in this PR relies on https://github.com/openshift/openshift-docs/pull/80042/files#diff-b50464e2135a901eea34a6c13abbd166151e637ba983e066f2297c5e9e8c313b, so waiting on merge for that first.

**Pull request merged**